### ISSUE TITLE
It only finds one plugin

### ISSUE THREAD

**Issue by chiahsien** [nrbrook/MakeXcodePluginsWork#5]
I have lots of plugins installed, and this script can find them all before I update to latest commit.
Now it can only find one plugin.

```
1) /Users/Nelson/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin
2) Back
3) Quit
```

**Comment by nrbrook** [comment]
What are the paths of the plugins it doesn't find?

**Comment by chiahsien** [comment]
I've tried 0e6be739c31ba98563f1b17f577004ad95d7064d, it works fine. 

```
 1) ./Alcatraz.xcplugin
 2) ./AMMethod2Implement.xcplugin
 3) ./AutoHighlightSymbol.xcplugin
 4) ./Cichlid.xcplugin
 5) ./DXXcodeConsoleUnicodePlugin.xcplugin
 6) ./HTYCopyIssue.xcplugin
 7) ./IntelliPaste.xcplugin
 8) ./LSLog-XCode.xcplugin
 9) ./OMColorSense.xcplugin
10) ./ProjectWindowName.xcplugin
11) ./SCXcodeSwitchExpander.xcplugin
12) ./SCXcodeTabSwitcher.xcplugin
13) ./UncrustifyPlugin.xcplugin
14) ./XcodePlus-DeleteLine.xcplugin
15) ./XReset.xcplugin
16) Back
17) Quit
```

But 12d0720ea013c1ce3d54ec71633b210bac1a4a4e doesn't.

```
1) /Users/Nelson/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin
2) Back
3) Quit
```

**Comment by nrbrook** [comment]
Fixed in 3bcaa8e44e0f25fff126395e082e1b629983ea92
